tools/test/test-setup.sh

```diff
@@ -236,6 +234,15 @@ else
   TEST_PATH="$(rlocation $TEST_WORKSPACE/$EXE)"
 fi
 
+# Redefine rlocation to notify users of its removal - it used to be exported.
+# TODO: Remove this before Bazel 9.
+function rlocation() {
```

Comment:
This doesn't seem to be compatible with `/bin/sh` which caused a test failure at https://buildkite.com/bazel/google-bazel-presubmit/builds/87576#0193afe3-eb2a-49df-b687-13d1be1ce0f7

I can change `#!/bin/sh`  to `#!/usr/bin/env bash` to fix it in pywrap_template.txt, but maybe we should try to make it compatible?

Reply:
```
function rlocation() {
  caller 0 | {
    read LINE SUB FILE
    echo >&2 "ERROR: rlocation is no longer implicitly provided by Bazel's test setup, but called from $SUB in line $LINE of $FILE. Please use https://github.com/bazelbuild/rules_shell/blob/main/shell/runfiles/runfiles.bash instead."
    exit 1
  }
}
```
seems to work